packages/extra/etcd/templates/hook/job.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Chart.Version | semverCompare ">= 2.6.1" }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: etcd-hook-pre
+  annotations:
+    helm.sh/hook: pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+spec:
+  template:
+    metadata:
+      labels:
+        policy.cozystack.io/allow-to-apiserver: "true"
+    spec:
+      serviceAccountName: etcd-hook
+      containers:
+        - name: helm
+          image: alpine/k8s:1.31.6
```

Comment:
Please use `bitnami/kubectl:latest` as we already consume it from in other hooks